src/renderer/i18n/en.json

```
@@ -406,6 +406,9 @@
     },
     "firmwareSources": "Source code",
     "calloutTitle": "Important!",
+    "factoryResetTitle": "Factory reset",
+    "factoryResetDescription": "Erase all settings layouts on your keyboard, restoring it to factory defaults.",
```

```suggestion
    "factoryResetDescription": "Erase all customizations from your keyboard, restoring it to factory defaults.",
```

Not sure about the suggested wording either, but "all settings layouts" doesn't sound right to me.